Engine/src/sh_field.cpp

```diff
@@ -314,6 +321,31 @@ void SHField::setFadeIfHidden(bool previous, bool fadeEnabled)
     }
 }
 
+void SHField::setVisibleSlices(State::CameraMode previous, State::CameraMode next)
+{
+    if(previous != next)
+    {
+        glm::ivec4 isVisible;
+        switch(next)
+        {
+            case State::CameraMode::projectiveX:
+                isVisible = glm::ivec4(1, 0, 0, 0);
+                break;
+            case State::CameraMode::projectiveY:
+                isVisible = glm::ivec4(0, 1, 0, 0);
+                break;
+            case State::CameraMode::projectiveZ:
+                isVisible = glm::ivec4(0, 0, 1, 0);
+                break;
+            case State::CameraMode::projective3D:
+            default:
+                isVisible = glm::ivec4(1, 1, 1, 0);
+                break;
+        }
+        mGridInfoData.Update(2*sizeof(glm::ivec4), sizeof(glm::ivec4), &isVisible);
```

Comment:
In function scaleSpheres you made this change do we also want to do this here?
```suggestion
        mGridInfoData.Update(3*sizeof(glm::ivec4), sizeof(glm::ivec4), &isVisible);
```